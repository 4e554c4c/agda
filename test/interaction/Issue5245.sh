#!/usr/bin/env bash

AGDA=${1}

cd Issue5245

cp safe lib-a.agda-lib
cp b/safe b/lib-b.agda-lib
cp c/safe c/lib-c.agda-lib

${AGDA} --interaction --no-default-libraries --library-file=libs <<EOF
IOTCM "LibA.agda" None Indirect (Cmd_load "LibA.agda" ["--ignore-interfaces"])
EOF

cp no-safe lib-a.agda-lib
cp b/no-safe b/lib-b.agda-lib
cp c/no-safe c/lib-c.agda-lib

rm c/_build/*/agda/LibC.agdai

${AGDA} --interaction --no-default-libraries --library-file=libs <<EOF
IOTCM "LibA.agda" None Indirect (Cmd_load "LibA.agda" [])
EOF

rm -r _build b/_build c/_build
rm lib-a.agda-lib b/lib-b.agda-lib c/lib-c.agda-lib
