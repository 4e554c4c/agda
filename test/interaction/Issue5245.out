(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" "Checking LibA (Issue5245/LibA.agda). " t)
(agda2-info-action "*Type-checking*" " Checking LibB (Issue5245/b/LibB.agda). " t)
(agda2-info-action "*Type-checking*" " Checking LibC (Issue5245/c/LibC.agda). " t)
(agda2-status-action "Checked")
(agda2-info-action "*All Done*" "" nil)
((last . 1) . (agda2-goals-action '()))
(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Type-checking*" " Checking LibC (Issue5245/c/LibC.agda). " t)
(agda2-info-action "*Type-checking*" " Checking LibB (Issue5245/b/LibB.agda). " t)
(agda2-info-action "*Type-checking*" "Checking LibA (Issue5245/LibA.agda). " t)
(agda2-status-action "Checked")
(agda2-info-action "*All Done*" "" nil)
((last . 1) . (agda2-goals-action '()))
